src/tests/tablet/schedule/CannotCreateMeetingWithStartTimeLessOfEndTime.java

```diff
@@ -0,0 +1,46 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC48: Verify when put a startTime less to endTime a error message is displayed
+ * @author Asael Calizaya
+ *
+ */
+public class CannotCreateMeetingWithStartTimeLessOfEndTime {
```

Comment:
The title say that a error message is displayed but the name of the class is different ....please check this inconsistent in your titles ..
It should be AMessageErrorIsDisplayedWhenStartTimeIsLessThanEndTime